pkg/toolsets/lvms/health.go

```
@@ -0,0 +1,271 @@
+		Alerts:    alerts,
+		Healthy:   len(alerts) == 0,
+	}
+
+	// Format output
+	var sb strings.Builder
+	sb.WriteString("# LVMS Thin Pool Health Check\n\n")
+
+	if len(thinPools) == 0 {
+		sb.WriteString("No thin pools found on any node.\n")
+	} else {
+		sb.WriteString(fmt.Sprintf("**Threshold:** %d%%\n", threshold))
```

golangci-lint (2.12.2)

[error] QF1012: Use fmt.Fprintf(...) instead of WriteString(fmt.Sprintf(...))

(staticcheck)